src/api/tests/createCustomer.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { test } from "fixtures/api-services.fixture";
+import { STATUS_CODES } from "data/statusCodes";
+import { generateCustomerData } from "data/customers/generateCustomer.data";
+import { TAGS } from "data/testTags.data";
+import { validateResponse } from "utils/validations/responseValidation";
+
+import { positiveTestCases, negativeTestCases } from "data/customers/customerTestCases.data";
+
+test.describe("[API] [Customers] Positive Tests", () => {
+
+    let token = "";
+    let id = "";
+
+    test.beforeEach(async ({ signInApiService }) => {
```

Comment:
Возможно достаточно один раз залогиниться и этого хватит для прогона набора
```suggestion
    test.beforeAll(async ({ signInApiService }) => {
```